Cython/Includes/cpython/unicode.pxd

```diff
@@ -23,6 +29,21 @@ cdef extern from *:
     # New in version 3.3.
     Py_ssize_t PyUnicode_GET_LENGTH(object o)
 
+    Py_UCS1 *PyUnicode_1BYTE_DATA(object o)
+    Py_UCS2 *PyUnicode_2BYTE_DATA(object o)
+    Py_UCS4 *PyUnicode_4BYTE_DATA(object o)
+
+    int PyUnicode_WCHAR_KIND  # Deprecated since version 3.10, will be removed in version 3.12.
```

Comment:
Let's not require us to change this line later just because the release of Python 3.12 goes as planned.
```suggestion
    int PyUnicode_WCHAR_KIND  # Deprecated since Python 3.10, removed in 3.12.
```